modules/fedimint-mint-client/src/lib.rs

```diff
@@ -248,6 +248,11 @@ impl MintClientExt for Client {
             federation_id,
             notes,
         } = oob_notes;
+        ensure!(
```

Comment:
Should we reject these on deserialization level?

Reply:
Maybe? It just seemed more natural to do it here because at least generating them has to be prevented in the client impl. I could add a second line of defense on the decoding level I guess.

Reply:
Addressed in [48a8125](https://github.com/fedimint/fedimint/pull/3281/commits/48a8125d2b26600cbe0903f5d9e1e393c936026b)